app/views/pets/show.html.erb

```diff
@@ -8,7 +8,7 @@
 <%= link_to 'Update Pet', "/pets/#{@pet.id}/edit", method: :get %>
 
 <% if session[:favorites] && session[:favorites].include?(@pet.id.to_s) %>
-  <% # Delete pet from favorites button goes here %>
+  <% # Delete pet from favorites button goes here* %>
```

Comment:
I looked over the specs and I didnt see anything about a remove from favorites button or link going on a pets show page.

Reply:
I looked into this. User story 12 says there should be a delete button. Thank you for bring this up because I accidentally attached the user story 12 issue to i2-session-refactor thinking it was completed and it is not. I will reopen user story 12 and complete it tomorrow. 

As a visitor
After I've favorited a pet
**When I visit that pet's show page
I no longer see a link to favorite that pet
But I see a link to remove that pet from my favorites**
When I click that link
A delete request is sent to "/favorites/:pet_id"
And I'm redirected back to that pets show page where I can see a flash message indicating that the pet was removed from my favorites
And I can now see a link to favorite that pet
And I also see that my favorites indicator has decremented by 1

Reply:
Oh ok, I was wondering about that because it would make sense to have the button